.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+       - image: circleci/ruby:2.4.1-node-browsers
+       - image: luckyj5/splunk-circleci-hec-720:latest
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run:
+          name: install dependencies
```

Comment:
"Install dependencies" this needs to be done for all the steps and i would prefer no hyphens in steps names